### PR TITLE
SME selection updates

### PR DESCRIPTION
**Pull request by wslack**

### CODE REVIEW

**Review by amy-usds — COMMENTED** [review]
Overall this is fine to merge, please address the issue of what happens if there is no diversity within the job currently (e.g., bring in SMEs from other agencies or outside consultative help).

File: _includes/_sme-selection-checklist.html [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 </p>
 
 <p>
-Agencies that are posting positions spanning multiple offices should have a wide and diverse range of SMEs present for job analysis.
+SMEs should reflect a representative sample of the job including demographics (e.g., age, gender, race and ethnicity, veteran status), location, organization (e.g., department if applicable, agency, office, work unit), and so forth. Lack of representation will make assessments less effective for fair and equitable hiring, and can impact the defensibility of your assessment strategy.
+
+You especially need a wide and diverse range of SMEs present for job analysis. Potential SMEs should complete a background information form and submit a resume as part of the selection process. Document this information in the case file for the hiring action.
```
Comment (wslack) [inline _includes/_sme-selection-checklist.html:9]:
```suggestion
SMEs should reflect a representative sample of the job including demographics (e.g., age, gender, race and ethnicity, veteran status), location, organization (e.g., department if applicable, agency, office, work unit), and so forth. Lack of representation will make assessments less effective for fair and equitable hiring, and can impact the defensibility of your assessment strategy. You especially need a wide and diverse range of SMEs present for job analysis, and should avoid situations where any SME is the sole voice on an important topic. If you aren't able to get a representative sample from existing personnel, considering bringing in SMEs from other agencies doing aligned work or consulting with outside resources or experts.

Potential SMEs should complete a background information form as part of the selection process (you can also take SME resumes). Document this information in the case file for the hiring action. 
```

Comment (wslack) [reply _includes/_sme-selection-checklist.html:9]: @amy-usds how is this?

**Review by amy-usds — APPROVED** [review]
Good to go.

**Pull request merged**